src/modules/endpoint/sagas/loadCityContent.js

```diff
@@ -79,7 +79,7 @@ export default function * loadCityContent (
 
     const fetchMap = resourceURLFinder.buildFetchMap(
       categoriesMap.toArray().concat(events),
-      (url, path, urlHash) => buildResourceFilePath(url, newCity, urlHash)
+      (url, urlHash) => buildResourceFilePath(url, newCity, urlHash)
```

Comment:
Hmmm... `buildResourceFilePath` needs 4 arguments; `path` is one of them...

Reply:
Not really? `export default (urlString: string, city: string, urlHash: string) => {
  return `${getResourceCacheFilesDirPath(city)}/${urlHash}${getExtensionWithDot(urlString)}`
}`

Reply:
Ok, I didn't see, that you're branching off of NATIVE-328.